src/client-ducky/src/components/Search.js

```diff
@@ -4,29 +4,56 @@ import React, {useState, useEffect} from 'react';
 import api from '../auth/api'
 
 
-const Search = () => {
+const sData = [
+    {
+        pk: 1,
+        title: "The best parking spot for TailGaiters",
+        desc: "If you are a tailgater, get this spot",
+        parking_size: "Tailgater Paradise",
+        price: 45.0
+    },
+    {
+        pk: 2,
+        title: "Cheap Motorcycle Spot",
+        desc: "Super simple spot, safe for long storage of motorcycle",
+        parking_size: "Motorcycle",
+        price: 4.0
+    },
+    {
+        pk: 3,
+        title: "Aggie Parking lot",
+        desc: "This is the parking lot between the spectrum and the football stadium",
+        parking_size: "Standard",
+        price: 7.5
+    },
+    {
+        pk: 4,
+        title: "Drive Way Close to Stadium",
+        desc: "Park on the left side of driveway for the game. Close, quick, convenient, cheap!",
+        parking_size: "Standard",
+        price: 6.0
+    }
+]
 
-    const [parkingSpots, setParkingSpots] = useState([]);
```

Comment:
why was this removed? Keeping it in the url params makes things much less global